iron-icon.html

```diff
@@ -86,18 +86,18 @@
   <template>
     <style>
       :host {
-        @apply(--layout-inline);
-        @apply(--layout-center-center);
+        @apply --layout-inline;
+        @apply --layout-center-center;
         position: relative;
 
         vertical-align: middle;
 
-        fill: var(--iron-icon-fill-color, currentcolor);
+        fill: var(--iron-icon-fill-color, currentColor);
```

Comment:
Looks like this change isn't necessary and `currentcolor` (w/o caps) might be the canonical spelling now, given [the note in this spec section](https://drafts.csswg.org/css-color/#currentcolor-color).